### ISSUE TITLE
FR: The seat locator should prioritize under extra seats

### ISSUE THREAD

**Issue by samip5** [Chicken/VenaaRauhassa#3]
As titled, it should be preferred if that's possible to do. 

Eg usually extra is wagon 2, upper part of it so wagon 2 downstairs should be preferred.

**Comment by Chicken** [comment]
Afaik as the extra wagon it has better internet connectivity.

**Comment by Chicken** [comment]
Yeah... the seat prioritization is very opinionated and to make it work properly it should have proper customization options. Haven't had motivation to think about how to implement it properly. Honestly should probably remove the feature from the site for the time being.
